app/connector/twitter/src/main/resources/META-INF/syndesis/connector/twitter.json

```diff
@@ -139,12 +139,14 @@
     "consumerKey": {
       "componentProperty": true,
       "deprecated": false,
-      "displayName": "Consumer Key",
+      "displayName": "Consumer API Key",
       "group": "security",
       "javaType": "java.lang.String",
       "kind": "property",
       "label": "security",
-      "labelHint": "The consumer key",
+      "labelHint": "The Consumer API key that Twitter provides when you register a client application.
+
+",
```

Comment:
These newlines cause the build to fail, can you please put the quote on the same line as the `labelHit` property.